bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/BagsResult.scala

```diff
@@ -0,0 +1,70 @@
+package edu.gemini.spModel.target.env
+
+import edu.gemini.shared.util.immutable.{Option => GOption}
+import edu.gemini.spModel.obs.context.ObsContext
+import edu.gemini.spModel.pio.{Pio, PioFactory, ParamSet}
+import edu.gemini.spModel.target.SPTarget
+import edu.gemini.spModel.rich.shared.immutable._
+
+sealed trait BagsResult extends Cloneable {
+  val id: String
+  val observationHash: String = ""
```

Comment:
What's the role of `observationHash`? It doesn't seem to be overidden on the derived classes but it is used in equals